deblur/deblurring.py

```diff
@@ -161,6 +162,6 @@ def deblur(input_seqs, read_error=0.05, mean_error=None, error_dist=None,
             # met all the criteria - so correct the frequency of the neighbor
             seq_j.frequency -= correction_value
 
-    result = [s for s in seqs if s.frequency > 0]
+    result = [s for s in seqs if round(s.frequency) > 0]
```

Comment:
We need to throw away sequences that are close to 0 (since at the end the read data is discrete). so we get read of sequences that after deblurring have <0.5 read left. Note that otherwise we would get 0 in the seq.to_fasta